### DISCUSSION TITLE
M64Plus Fz Pro - Redmi Note 11 Black Screen Netplay

### DISCUSSION THREAD

**Discussion by oliveiraren** [mupen64plus-ae/mupen64plus-ae#1124] [General]
I bought the Pro version, but the multiplayer mode has a problem. When I try create netplay server and start the game, the screen stay black and not load the game.

**Comment by fzurita** [comment]
This is a known issue and unfortunately I don't have the affected devices so it's hard to fix without that.

Does it happen if you use different emulation profiles?

**Reply by Nimrod777** [reply]
Tengo el mismo problema, mi celular es un Redmi 9 con MIUI 13.0.2

**Comment by oliveiraren** [comment]
Yes. This happens even when changing the emulation profile. But if you're working on fix, I'll wait until a future fix.
